database/migrations/2022_11_01_163211_create_beverages_table.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class () extends Migration {
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('beverages', function (Blueprint $table) {
+            $table->id();
+            $table->timestamps();
+            $table->string('name');
+            $table->string('brewer');
+            $table->string('abv');
+            $table->string('type');
+            $table->string('country')->nullable();
+            $table->string('region')->nullable();
+            $table->string('style')->nullable();
+            $table->string('producer')->nullable();
+        });
+
+        Schema::create('venue_beverage', function (Blueprint $table) { // still have an id column
```

Comment:
Iv been having issues when I call my link table venue_attribute but attribute_venue works.

So I wonder if this should be called beverage_venue?

Reply:
hmm interesting ill have a better look around this in the evening.